### ISSUE TITLE
failing tests

### ISSUE THREAD

**Issue by mcepl** [acoular/acoular#55]
While packaging this for openSUSE/Factory I have four tests (well, it is actually one test) failing:

```
[  111s] ======================================================================
[  111s] FAIL: test_beamformer_freq_results (test_beamformer_results.acoular_beamformer_test) [BeamformerCMF global_caching = none]
[  111s] ----------------------------------------------------------------------
[  111s] Traceback (most recent call last):
[  111s]   File "/home/abuild/rpmbuild/BUILD/acoular-21.05/acoular/tests/test_beamformer_results.py", line 91, in test_beamformer_freq_results
[  111s]     np.testing.assert_allclose(actual_data, ref_data, rtol=1e-5, atol=1e-8)
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 1530, in assert_allclose
[  111s]     assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
[  111s]     raise AssertionError(msg)
[  111s] AssertionError:
[  111s] Not equal to tolerance rtol=1e-05, atol=1e-08
[  111s]
[  111s] Mismatched elements: 81 / 338 (24%)
[  111s] Max absolute difference: 0.00439613
[  111s] Max relative difference: 55.84388
[  111s]  x: array([[[-4.597212e-04,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,...
[  111s]  y: array([[[0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,...
[  111s]
[  111s] ======================================================================
[  111s] FAIL: test_beamformer_freq_results (test_beamformer_results.acoular_beamformer_test) [BeamformerCMF global_caching = none]
[  111s] ----------------------------------------------------------------------
[  111s] Traceback (most recent call last):
[  111s]   File "/home/abuild/rpmbuild/BUILD/acoular-21.05/acoular/tests/test_beamformer_results.py", line 100, in test_beamformer_freq_results
[  111s]     np.testing.assert_allclose(actual_data, ref_data, rtol=1e-5, atol=1e-8)
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 1530, in assert_allclose
[  111s]     assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
[  111s]     raise AssertionError(msg)
[  111s] AssertionError:
[  111s] Not equal to tolerance rtol=1e-05, atol=1e-08
[  111s]
[  111s] Mismatched elements: 81 / 338 (24%)
[  111s] Max absolute difference: 0.00439613
[  111s] Max relative difference: 55.84388
[  111s]  x: array([[[-4.597212e-04,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,...
[  111s]  y: array([[[0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,...
[  111s]
[  111s] ======================================================================
[  111s] FAIL: test_beamformer_freq_results (test_beamformer_results.acoular_beamformer_test) [BeamformerCMF global_caching = none]
[  111s] ----------------------------------------------------------------------
[  111s] Traceback (most recent call last):
[  111s]   File "/home/abuild/rpmbuild/BUILD/acoular-21.05/acoular/tests/test_beamformer_results.py", line 109, in test_beamformer_freq_results
[  111s]     np.testing.assert_allclose(actual_data, ref_data, rtol=1e-5, atol=1e-8)
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 1530, in assert_allclose
[  111s]     assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
[  111s]     raise AssertionError(msg)
[  111s] AssertionError:
[  111s] Not equal to tolerance rtol=1e-05, atol=1e-08
[  111s]
[  111s] Mismatched elements: 81 / 338 (24%)
[  111s] Max absolute difference: 0.00439613
[  111s] Max relative difference: 55.84388
[  111s]  x: array([[[-4.597212e-04,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,...
[  111s]  y: array([[[0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,...
[  111s]
[  111s] ======================================================================
[  111s] FAIL: test_beamformer_freq_results (test_beamformer_results.acoular_beamformer_test) [BeamformerCMF global_caching = overwrite]
[  111s] ----------------------------------------------------------------------
[  111s] Traceback (most recent call last):
[  111s]   File "/home/abuild/rpmbuild/BUILD/acoular-21.05/acoular/tests/test_beamformer_results.py", line 124, in test_beamformer_freq_results
[  111s]     np.testing.assert_allclose(actual_data, ref_data, rtol=1e-5, atol=1e-8)
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 1530, in assert_allclose
[  111s]     assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
[  111s]   File "/usr/lib64/python3.9/site-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
[  111s]     raise AssertionError(msg)
[  111s] AssertionError:
[  111s] Not equal to tolerance rtol=1e-05, atol=1e-08
[  111s]
[  111s] Mismatched elements: 81 / 338 (24%)
[  111s] Max absolute difference: 0.00439613
[  111s] Max relative difference: 55.84388
[  111s]  x: array([[[-4.597212e-04,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,
[  111s]           0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,...
[  111s]  y: array([[[0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,
[  111s]          0.000000e+00, 0.000000e+00, 0.000000e+00, 0.000000e+00,...
[  111s]
[  111s] ----------------------------------------------------------------------
[  111s] Ran 20 tests in 93.292s
[  111s]
```

[Complete build log](https://github.com/acoular/acoular/files/8154051/_log.txt) with all packages versions and steps taken.

**Comment by esarradj** [comment]
This may be related to incompatible changes in scitkit-kearn >= 1.0
I'll look into it.

**Comment by esarradj** [comment]
Problem is fixed in the upcoming release.

**Comment by esarradj** [comment]
Fixed in 22.03
